t/27-consoles-sshIucvconn.t

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/perl
+
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::sshIucvconn;
+use Test::MockObject;
+use Test::MockModule;
+use testapi;
+
+my $testapi_console = 'sshIucvconn';
+my $args = { hostname => 'testhost', password => 'testpass' };
+my $exec_flag = 1;
+$bmwqemu::vars{ZVM_GUEST} = 'guest.what';
+my $backend_mock = Test::MockObject->new();
+
+$backend_mock->mock('new_ssh_connection', sub($self, %args) {
+    my $ttyconn_mock = Test::MockObject->new;
+    $ttyconn_mock->mock('channel', sub {
+        my $channel_mock = Test::MockObject->new;
+        $channel_mock->mock('blocking', sub { 1 });
+        $channel_mock->mock('pty', sub { 1 });
+        $channel_mock->mock('exec', sub { $exec_flag });
+        return $channel_mock; });
```

Comment:
First, you can chain calls

```suggestion
        return $channel_mock->mock('blocking', sub { 1 })->mock('pty', sub { 1 })->mock('exec', sub { $exec_flag });
```

and second no need to use the generic `mock` when you just want to return a value

```suggestion
        return $channel_mock->set_true('blocking', 'pty')->set_always('exec', $exec_flag);
```

and then it's a common style to use the big comma and leave out the single ticks

```suggestion
        return $channel_mock->set_true('blocking', 'pty')->set_always(exec => $exec_flag);
```

Reply:
You provided a thumbs up but still didn't include the suggestion? Please check your latest state of code again. And to prevent too many comments going back and forth I recommend that you carefully check your actual state of code if you actually encorporated the expected changes before pushing which triggers reviewers for another round of review.

Reply:
I chain the mocks. but the return isnt needed.

Reply:
Please take a look into my code suggestion. You still haven't used it but your latest comment states that you would agree.